compiler/optimizer/src/main/java/org/apache/nemo/compiler/optimizer/NemoOptimizer.java

```diff
@@ -46,6 +48,8 @@
 public final class NemoOptimizer implements Optimizer {
   private final String dagDirectory;
   private final Policy optimizationPolicy;
+  private final String environmentType;
```

Comment:
`private final EnvironmentType environmentType`
or change this variable name to `environmentTypeStr`

Reply:
I've renamed the variable. Thanks!